spec/acceptance/suites/default/00_default_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'spec_helper_acceptance'
+
+test_name 'incron'
+
+describe 'incron class' do
+  let(:test_file_content) {
+    'this is a test'
+  }
+
+  let(:test_target) { '/tmp/test_output' }
+
+  let(:test_script) {[
+   %(#!/bin/sh),
+   %(echo $1 > #{test_target})
+  ].join("\n")}
+
+  let(:test_script_path) { '/root/test.sh' }
+
+  hosts.each do |host|
+    context "on #{host}" do
+      let(:manifest) {
+        <<-EOS
+          incron::system_table { 'puppet_test':
+            path    => "#{host.puppet[:environmentpath]}/*/modules/*/lib",
+            command => "#{test_script_path} $@/$#"
+          }
+        EOS
+      }
+      let(:test_file) {
+        "#{host.puppet[:environmentpath]}/production/modules/incron/lib/test"
+      }
+
+      it 'should work with default values' do
+        apply_manifest_on(host, manifest, catch_failures: true)
+      end
+
+      it 'should be idempotent' do
+        apply_manifest_on(host, manifest, catch_changes: true)
+      end
+
+      it 'should have a test script to run' do
```

Comment:
is there a reason this isn't just one it block, since the two steps are setting up the environment for the final test?

Reply:
Because having `should have a test script to run and create a file that triggers incron` seemed like a bit much for one line in the output.

Also, the first it test prep, the second is actually triggering an action on the system.